standalone_tests/all_media_types.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
are there any assertions in this test to check that data was handled correctly? or is this just testing that w&b doesn't crash on these inputs?

also this is very large which can make failures hard to investigate and fix. would it make sense to split this out into multiple smaller tests?

Reply:
I guess assertions wouldn't hurt, but this is meant to be a manual test to help us check that this functionality works from end-to-end. I prefer having it as one file so I can run it once and then check that everything works. It would definitely be good to turn this into a larger integration test but I think this is a good first step.

Reply:
Like, assertions and splitting things up won't help test frontend functionality. For me right now this is a manual integration test.